Tools/clinic/clinic.py

```diff
@@ -5514,7 +5528,7 @@ def state_modulename_name(self, line: str) -> None:
 
         self.function = Function(name=function_name, full_name=full_name, module=module, cls=cls, c_basename=c_basename,
                                  return_converter=return_converter, kind=self.kind, coexist=self.coexist,
-                                 critical_section=self.critical_section)
+                                 critical_section=self.critical_section, target_critical_section=self.target_critical_section)
```

Comment:
Let's wrap new code to ~90 characters per line where possible :)

```suggestion
                                 critical_section=self.critical_section,
                                 target_critical_section=self.target_critical_section)
```